SignalServiceKit/src/Messages/Interactions/TSInteraction.m

```diff
@@ -14,6 +14,8 @@ @implementation TSInteraction
 
 + (instancetype)interactionForTimestamp:(uint64_t)timestamp
                         withTransaction:(YapDatabaseReadWriteTransaction *)transaction {
+    OWSAssert(timestamp > 0);
```

Comment:
Some message types have a timestamp of zero (like group updates IIRC).

Reply:
Yup, this assert is meant to start catching those.

Reply:
I'm going to add another similar assert in the `TSInteraction` initializer to determine if this is indeed an issue.